plugins/nominal-connection-checker/src/hierarchy_validation.js

```diff
@@ -95,7 +95,7 @@ function checkSupersDefined(hierarchyDef) {
  * },
  * 'typeB': {
  *   'fulfills': ['typeA']
- * }
+ * }.
```

Comment:
This is a bit weird, but I guess it is just trying to fulfill the rule that comments should be a complete sentence?

Reply:
Yeah I'm not in love with it but it doesn't seem worth fighting it either.